### ISSUE TITLE
Dockerfile outdated

### ISSUE THREAD

**Issue by balazon** [NVlabs/instant-ngp#1406]
when I try to pull the base image, it fails:
`docker pull nvidia/cuda:10.2-devel-ubuntu18.04`
>> Error response from daemon: manifest for nvidia/cuda:10.2-devel-ubuntu18.04 not found: manifest unknown: manifest unknown

When checking dockerhub for official image:
https://hub.docker.com/r/nvidia/cuda/tags?page=1&name=10.2
there is no official 10.2 image by nvidia any more, the list is empty.

Can you update the Dockerfile with a newer base image?
I'm still testing it, but maybe 
nvidia/cuda:11.7.1-devel-ubuntu18.04
might work

Thanks

**Comment by BreakingPitt** [comment]
Hi:

Yes, I can confirm that the information provided by @balazon is correct!

It seems that the specific NVIDIA CUDA image tag defined, 10.2-devel-ubuntu18.04, is no longer available in Docker Hub.

This is why we are getting the "manifest unknown" error when trying to build the image.

**Comment by Tom94** [comment]
I believe docker images corresponding to newer CUDA versions should work out of the box. If someone has the time to try one, confirm it working and make a pull request I'd be happy to merge.

**Comment by Tom94** [comment]
Looks good to me. In the pull request, you're asking for someone to test -- I don't have a suitable setup to do this at the moment, so I will hold off from merging until someone confirmed it works.

**Comment by BreakingPitt** [comment]
@Tom94  Yeah, no problem... lets see if someone can confirm that it works as expected!
